cmd/pulumi-test-language/interface.go

```diff
@@ -1233,6 +1252,15 @@ func (eng *languageTestServer) RunLanguageTest(
 			}, nil
 		}
 
+		eventsCts = &promise.CompletionSource[[]engine.Event]{}
+		eventSink = make(chan engine.Event, 1)
```

Comment:
Does `eventSink` need to be passed to `s.Update`?

Reply:
🤦 yes

Reply:
Good prompt to make use of these events in an update test to ensure they're actually sent.